Content.Server/Projectiles/ProjectileSystem.cs

```diff
@@ -31,6 +32,14 @@ private void OnStartCollide(EntityUid uid, ProjectileComponent component, ref St
             || component.DamagedEntity || component is { Weapon: null, OnlyCollideWhenShot: true })
             return;
 
+        // Don't collide if the laying down entity is not our target.
+        if (TryComp<TargetedProjectileComponent>(uid, out var targetedProjectile))
+        {
+            if(args.OtherEntity != targetedProjectile.Target &&
+               args.OtherFixture.CollisionLayer == (int) CollisionGroup.LayingDownMobLayer)
+                return;
+        }
+
```

Comment:
Collision flags are bitmasks so checking exact equality shouldn't be done.

Reply:
Made it subtract/add Layers to laying down entities, Targeted projectiles now get a mask added.